### ISSUE TITLE
2 Requested Ports from BTTV

### ISSUE THREAD

**Issue by Jaladhjin** [FrankerFaceZ/FrankerFaceZ#572]
I really reeeeaaally miss Pinning Hightlighted Messages since if I'm not intently watching chat like a hawk & even in some fast moving chats I'd miss highlighted messages altogether.

And removing sub.. bits & basically all but actual chat related info from the chat.

I'd still see the subs come up on stream since.. can't really block them & that's cool ha.. but chat is for chat :-D

I'm aware I could try to use both together but that seems kinda silly like I'm asking for wonky conflicts.

Or trying to turn off 1 feature in 1 mod & on in the other.

I was a long time BTTV user but I'm a bigger fan of this for the most part.. nice work ^_^

**Comment by Tester798** [comment]
Yes, "Pinning Hightlighted Messages" is the feature from BTTV I'm missing the most, very helpful.
